ci/job.yml

```diff
@@ -0,0 +1,168 @@
+- job:
+    name: 'centos-container-pipeline-service-container-index'
```

Comment:
Is it just me or it's actually confusing that the name has both "Container Pipeline Service" and "Container Index"? Which one exactly does it do CI for?

Reply:
I think we could prepend our CI jobs with `ccp` if that's the reason why we have mentioned "centos-container-pipeline-service" in the name.

Reply:
updated to `centos-container-index-ci`.